android/src/main/java/vn/luongvo/kmm/survey/android/ui/theme/AppTypography.kt

```diff
@@ -0,0 +1,43 @@
+package vn.luongvo.kmm.survey.android.ui.theme
+
+import androidx.compose.material.Typography
+import androidx.compose.runtime.staticCompositionLocalOf
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.*
+import androidx.compose.ui.unit.sp
+import vn.luongvo.kmm.survey.android.R
+
+private val Neuzeit = FontFamily(
+    Font(R.font.neuzeit_book, FontWeight.Normal),
+    Font(R.font.neuzeit_heavy, FontWeight.Bold)
+)
+
+private val Typography = Typography(
+    defaultFontFamily = Neuzeit,
+    body1 = TextStyle(
+        fontSize = 17.sp,
+        fontWeight = FontWeight.Normal
+    ),
+    body2 = TextStyle(
+        fontSize = 15.sp,
+        fontWeight = FontWeight.Normal
+    ),
+    subtitle1 = TextStyle(
+        fontSize = 13.sp,
+        fontWeight = FontWeight.Bold
+    ),
+    h4 = TextStyle(
+        fontSize = 26.sp,
+        fontWeight = FontWeight.Bold
+    ),
+    h5 = TextStyle(
+        fontSize = 34.sp,
+        fontWeight = FontWeight.Bold
+    ),
```

Comment:
`h4` size should bigger than `h5` size IMO
```suggestion
    h4 = TextStyle(
        fontSize = 34.sp,
        fontWeight = FontWeight.Bold
    ),
    h5 = TextStyle(
        fontSize = 26.sp,
        fontWeight = FontWeight.Bold
    ),
```

Reply:
Fixed in https://github.com/luongvo/kmm-survey/pull/48/commits/2bf5858983ff990d311a04c1e83df985919b6207